Cargo.toml

```diff
@@ -13,4 +13,4 @@ itertools = "0.11.0"
 clap = { version = "4.4.18", features = ["derive"] }
 
 [profile.release]
-debug = true
+strip = true
```

Comment:
the `debug = true` shouldn't be removed if there is no reason for it. also see #134